src/pages/index.astro

```diff
@@ -51,11 +51,16 @@ import {COMMUNITY_INVITE_URL, GITHUB_ORG_URL, REDDIT_URL} from '../consts';
         </a> or contribute to our community projects on
         <a href={GITHUB_ORG_URL}
            target="_blank"
-           title="Check out our subreddit">
+           title="Check out our github">
             GitHub
         </a>!
     </p>
-    <img src="/invite.png" alt="Search in the documentation">
+    <br style="margin-top: 0.5px"/>
```

Comment:
Inline styles should be avoided at all costs. Could you please put this in the style sheet below?